pages/blog/posts/oracle-case-study.md

```diff
@@ -0,0 +1,1123 @@
+---
+title: "JSON Schema and Relational Databases: Bridging the Gap!"
+date: "2025-01-31"
+tags:
+  - database
+  - relational
+type: Case Study
```

Comment:
It might be just me, but I think the article goes beyond the typical case study (i.e. here is how company X uses JSON Schema) into actually teaching something to the reader. From that point of view, maybe better to post it on the "Engineering" category?

Reply:
On the other side, I think Oracle's case study is indeed very technical, so maybe that's fine. Not sure. @benjagm what do you think?